docs/content/getting-started/data-out.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Get data out of Rerun
+order: 700
+---
+
+At its core, Rerun is a database. The viewer includes the [dataframe view](../reference/types/views/dataframe_view) to explore data in tabular form, and the SDK includes an API to export data as dataframes from recording. These features can be used, for example, to perform analysis on the data and log back the results to the original recording.
```

Comment:
> At its core, Rerun is a database.

I get what you're getting at, but that's not really how we want to define ourselves though. Rerun Commercial will be a data platform. It's just that the viewer & rrd files act like databases 🤔 

I'll try some other formulation

Reply:
Discussed this internally a lil bit: Given that we _do_ have a [multimodal, ecs] database at the core _and_ we really want people to think about everything in this section about "the database" this formula makes a lot of sense

Reply:
I think saying Rerun is a database at its core is very in line with how we want to define ourselves. Partly because we're still trying to get sharper on exactly how to define ourself, there are currently multiple ways we describe Rerun. We are building a data platform that will feel very database-y and be very easy to define as a database (quacks like a duck etc). The one liner is that we are "the multimodal data stack for spatial and embodied AI". That's also reductive of course.

It's a very useful to think of Rerun as a database so I think this is a great formulation for this page